extensions/amp-render/1.0/amp-render.js

```diff
@@ -246,9 +247,49 @@ export class AmpRender extends BaseElement {
       this.mutateProps(
         dict({
           'render': (data) => {
+            if (this.element.getAttribute('binding') === 'no') {
+              return templates
+                .renderTemplateAsString(dev().assertElement(template), data)
+                .then((html) => dict({'__html': html}));
+            }
+
+            let elements;
+            if (this.element.getAttribute('binding') === 'refresh') {
+              return templates
+                .renderTemplate(dev().assertElement(template), data)
+                .then((el) => {
+                  elements = isArray(el) ? el : [el];
```

Comment:
In which cases do you get an array vs. an individual element from `renderTemplate`?

Reply:
(related nit: use `arrayOrSingleItemToArray` from `types/array` if still needed)

Reply:
it always returns a single element. no need for the check anymore.

Reply:
it always returns a single element. no need for the check anymore.